setup.py

```diff
@@ -89,57 +88,27 @@ def run(self):
     package_dir={'': 'src'},
     packages=find_packages('src'),
 
-    setup_requires=[
-        'pytest-runner',
-    ],
     install_requires=[
+        'apache-airflow',
+
+        'funcsigs; python_version<"3"',
         'jinja2>=2.8',
         'trafaret-config==1.0.1',
-    ] + (['funcsigs'] if sys.version_info[:2] == (2, 7) else []),
-    tests_require=[
-        'apache-airflow==1.8.1',
-        'pytest==3.1.3',
-        'pytest-cov==2.5.1',
-        'pytest-flake8==0.8.1',
-        'pytest-isort==0.1.0',
-        'pytest-sugar==0.8.0',
+        'trafaret<0.11',
     ],
     extras_require={
         'develop': [
-            'apache-airflow==1.8.1',
+            'flake8==3.5.0',
+            'isort==4.3.4',
             'mock==2.0.0',
-            'pytest==3.1.3',
-            'pytest-cov==2.5.1',
-            'pytest-flake8==0.8.1',
-            'pytest-isort==0.1.0',
-            'pytest-sugar==0.8.0',
+            'pylint==2.1.1; python_version>="3"',
```

Comment:
Can we have pylint checks for Python 2.x as well? I'm fine with negative answer, but just curious.

Reply:
I believe we can. I added this condition because that version of pylint is not available on 2.7. Apparently they have dropped support for 2.7 in the recent versions, so an older version of pylint should be used on 2.7. 

Furthermore running linters from both 3.6 and 2.7 seems like an overkill to me.

Reply:
Ok, let's avoid any non-simple work that requires for 2.7 support.